plugins/storage/volume/linstor/src/main/java/com/cloud/hypervisor/kvm/storage/LinstorStorageAdaptor.java

```diff
@@ -214,6 +215,7 @@ private void makeResourceAvailable(DevelopersApi api, String rscName, boolean di
     public KVMPhysicalDisk createPhysicalDisk(String name, KVMStoragePool pool, QemuImg.PhysicalDiskFormat format,
                                               Storage.ProvisioningType provisioningType, long size, byte[] passphrase)
     {
+        s_logger.debug("Linstor.createPhysicalDisk: " + name + ";" + format);
```

Comment:
use `String.format()`?
```suggestion
        s_logger.debug(String.format("Linstor.createPhysicalDisk: %s;%s", name, format));
```